extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -9,11 +9,20 @@ $color-borders: #CECECE;
 
 @import 'extensions/wikia/ThemeDesigner/css/ThemeTab';
 
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/_colors.scss';
+
+$normal-spacing: 24px;
+$cnw-size: 700px;
+
+.skin-oasis.page-Special_CreateNewWiki {
+	background-color: $wds-color-off-white;
+}
+
 .WikiaPage {
 	border: 0;
 
 	.WikiaPageBackground {
-		@include linear-gradient(false, top, #01386e, 0%, #2185da, 25%, #fff, 60%);
+		background-color: $wds-color-off-white;
```

Comment:
Please set background only in one place. If the background is inherited you can overwrite it by `background: none`;